apps/extension/src/features/accounts/account-card.tsx

```diff
@@ -0,0 +1,376 @@
+                                }}
+                              ></div>
+                            </span>
+                          </div>
+                        </RadioGroup.Indicator>
+                      </RadioGroup.Item>
+                    );
+                  })}
+
+                  {/* {isSelected && (
+                      <div className="absolute flex bottom-1 right-1">
+                        <span className="checkmark">
+                          <div
+                            className="checkmark_circle"
+                            style={{ backgroundColor: darkColor }}
+                          ></div>
+                          <div
+                            className="checkmark_stem"
+                            style={{ backgroundColor: color }}
+                          ></div>
+                          <div
+                            className="checkmark_kick"
+                            style={{ backgroundColor: color }}
+                          ></div>
+                        </span>
+                      </div>
+                    )} */}
+                </div>
+              </RadioGroup.Root>
+
+              {/* {updating && <Spinner />} */}
+              {/* </div> */}
+            </div>
+          </div>
+          {/* Actions */}
+          <DropdownMenuDemo account={account} container={container!} />
+        </div>
+        {/* <div>
+          <div className="flex items-center space-x-5">
+            {account.isPrimary && (
+              <Tooltip.Root delayDuration={300}>
+                <Tooltip.Trigger>
+                  <div className="text-white bg-[#002F76] text-xs px-2 py-1 rounded cursor-help">
+                    Organizer
+                  </div>
+                </Tooltip.Trigger>
+                <Tooltip.Content align="end" side="bottom" sideOffset={10}>
+                  <div
+                    className="px-5 py-6 bg-white rounded-lg shadow-2xl"
+                    style={{ width: "350px" }}
+                  >
+                    <div className="font-semibold">Organizer Account</div>
+                    <div className="pt-2">
+                      The selected account will be the organizer for all events
+                      created.
+                    </div>
+                  </div>
+                </Tooltip.Content>
+              </Tooltip.Root>
+            )}
+            {optionsDropdown}
+          </div>
+        </div> */}
+      </div>
+      <div ref={setContainer} className="relative z-10"></div>
+      {/* <Modal
+        open={showDeleteModal}
+        title="Are you sure you want to remove this calendar from AgreeTo?"
+        primaryButton={{
+          text: "Remove",
+          onClick: handleDelete,
+          type: "danger",
+        }}
+        cancelButton={{
+          text: "Cancel",
+          onClick: () => setShowDeleteModal(false),
+        }}
+      /> */}
```

Comment:
Does it not work with the Modal here, or why is it commented out? would clean up quite a bit

Reply:
I wanted to use radix, which have an `AlertDialog` for this.

I can move that to `ui` though